packages/contracts/test/contracts/L1/rollup/CanonicalTransactionChain.gas.spec.ts

```diff
@@ -240,23 +245,25 @@ describe('[GAS BENCHMARK] CanonicalTransactionChain', () => {
       })
 
       const receipt = await res.wait()
+      const gasUsed = receipt.gasUsed.toNumber()
 
       console.log('Benchmark complete.')
-      console.log('Gas used:', receipt.gasUsed.toNumber())
+      console.log('Gas used:', gasUsed)
+
       console.log('Fixed calldata cost:', fixedCalldataCost)
       console.log(
         'Non-calldata overhead gas cost per transaction:',
-        (receipt.gasUsed.toNumber() - fixedCalldataCost) / numTxs
+        (gasUsed - fixedCalldataCost) / numTxs
       )
+      expectApprox(gasUsed, 1_126_553, { upperPercentDeviation: 0 })
```

Comment:
I assume it's intentional to have a bound of 0% but just confirming

Reply:
yep. even 1% is a lot in this case.